lib/oppen/utils.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+# Oppen.
+module Oppen
```

Comment:
This kind of "documentation" seems pretty useless.

Reply:
Since rubocop requires all modules to be documented at least once, I added these for consistency in the files. If they bother you I can remove them and only keep the one in the oppen.rb file

Reply:
Yes, just once is good.